src/app/list-of-movies/list-of-movies.component.ts

```diff
@@ -0,0 +1,22 @@
+import { MoviesService, Token, Watchlist } from './../movies.service';
+import { Observable } from 'rxjs/internal/Observable';
+import { environment } from './../../environments/environment';
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-list-of-movies',
+  templateUrl: './list-of-movies.component.html',
+  styleUrls: ['./list-of-movies.component.css']
+})
+export class ListOfMoviesComponent implements OnInit {
+  constructor(
+    private http: HttpClient,
+    private moviesService: MoviesService
+  ) { }
+  watchlist$: Observable<Watchlist> | undefined
```

Comment:
Zamiast robić dodatkowy interfejs 
```
export interface Watchlist {
  results: Array<WatchlistResult>
}
```
lepiej tu zrobić `Observable<Array<Watchlist>>`. Wtedy w html nie musisz robić ngIf tylko sam ngFor
```
<div *ngFor="let watchlistElement of watchlist$ | async">
```
Pozatym to bardziej wygląda jak film zapisany w "watchlist" więc raczej szedłbym w kierunku nazwy "watchlistMovies"